proto/doc.go

```diff
@@ -39,7 +39,73 @@
 //	                    |
 //	         ======= Dataplane ==========
 //
-// Overview
+//
+// Data-model Overview
+//
+// The data-model used on the dataplane driver API uses similar concepts to
+// the main "datastore" data-model (such as host/workload endpoints, policies
+// and profiles).  However, the calculation engine does some pre-calculation
+// that simplifies the job of the dataplane driver:
+//
+// Rules in the datastore data-model can contain selectors, such as
+// "role == webserver", that refer to dynamic groups of endpoints.  The calculation
+// engine pre-renders those selectors (and the older tag mechanism) to sets of
```

Comment:
'compute' would be clearer to me than 'render'.  (+ another occurrence following)